modules/installation-aws-config-yaml.adoc

```diff
@@ -301,15 +305,13 @@ accounts for the dramatically decreased machine performance. Use larger
 instance types, such as `m4.2xlarge` or `m5.2xlarge`, for your machines if you
 disable simultaneous multithreading.
 ====
-<6> To configure faster storage for etcd, especially for larger clusters, set the
-storage type as `io1` and set `iops` to `2000`.
+<6> To configure faster storage for etcd, especially for larger clusters, set the storage type as `io1` and set `iops` to `2000`.
 ifdef::vpc,restricted[]
 <7> If you provide your own VPC, specify subnets for each availability zone that your cluster uses.
-<8> The ID of the AMI used to boot machines for the cluster. If set, the AMI
-must belong to the same region as the cluster.
-<9> The AWS service endpoints. Custom endpoints are required when installing to
-an unknown AWS region. The endpoint URL must use the `https` protocol and the
-host must trust the certificate.
+<8> The ID of the AMI used to boot machines for the cluster. If set, the AMI must belong to the same region as the cluster.
+<9> The AWS service endpoints. Custom endpoints are required when installing to an unknown AWS region. The endpoint URL must use the `https` protocol and the host must trust the certificate.
++
+If you are installing to the Secret Commercial Cloud Services (SC2S) Region, specifying the Route53 endpoint is required.
```

Comment:
Similar comment as above, this should not be required for sc2s after https://issues.redhat.com/browse/CORS-1896

Reply:
Done. Thanks.